Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "QuizTrain",
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "QuizTrain",
+            targets: ["QuizTrain"]),
```

Comment:
Do you intend for this to be linked into multiple targets in the future? If so, please apply the following suggestion.

```suggestion
        .library(
            name: "QuizTrain",
            type: .dynamic,
            targets: ["QuizTrain"]),
```

Reply:
i'll incorporate it as it might end up linked into multiple targets but we have different plans for the repo.